.eslintignore

```diff
@@ -8,17 +8,16 @@
 **/semver/**
 **/test/**/*.js
 **/node_modules/**
-**/vscode-api-tests/testWorkspace/**
-**/vscode-api-tests/testWorkspace2/**
-**/extensions/**/out/**
-**/extensions/**/build/**
-**/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts
-**/big-data-cluster/src/bigDataCluster/controller/clusterApiGenerated2.ts
-**/extensions/markdown-language-features/media/**
-**/extensions/markdown-language-features/notebook-out/**
-**/extensions/typescript-basics/test/colorize-fixtures/**
-**/extensions/**/dist/**
-**/extensions/typescript-language-features/test-workspace/**
+/extensions/**/out/**
```

Comment:
Also needed to add skipping the types folder, and while I was here cleaned up these paths (it doesn't make sense to have ** before the extensions folder for these - it's an absolute path from the top)